app/modules/sightings/models.py

```diff
@@ -21,6 +28,8 @@ class Sighting(db.Model, FeatherModel):
     )  # pylint: disable=invalid-name
     version = db.Column(db.BigInteger, default=None, nullable=True)
 
+    assets = db.relationship('SightingAssets')
```

Comment:
Is this necessary? Surely this is created by line 17 above with the back_populates?

Reply:
Well, I wouldn't have though so but sqlalchemy seems to think it is, even with a fresh migration:
`sqlalchemy.exc.InvalidRequestError: Mapper 'mapped class Sighting->sighting' has no property 'assets'`